src/aerospike_clj/client.clj

```diff
@@ -518,23 +549,26 @@
 
 (defn healthy?
   "Returns true iff the cluster is reachable and can take reads and writes.
-  Uses __health-check set to avoid data collisions. `operation-timeout-ms` is for total timeout of reads
-  (including 2 retries) so an small over estimation is advised to avoid false negatives."
-  [db operation-timeout-ms]
-  (let [read-policy (let [p (.readPolicyDefault ^AerospikeClient (get-client db ""))]
-                      (set! (.totalTimeout p) operation-timeout-ms)
-                      p)
-        k (str "__health__" (rand-int 1000))
-        v 1
-        ttl (min 1 (int (/ operation-timeout-ms 1000)))
-        set-name "__health-check"]
-    (try
-      @(create db k set-name v ttl)
-      (= v
-         @(get-single db k set-name {:transcoder :payload
-                                     :policy read-policy}))
-      (catch Exception _ex
-        false))))
+  Uses __health-check set to avoid data collisions. `operation-timeout-ms` is
+  for total timeout of reads (default is 1s) including 2 retries so a small
+  over estimation is advised to avoid false negatives."
+  ([db]
+   (healthy? db 1000))
+  ([db operation-timeout-ms]
+   (let [read-policy (let [p (.readPolicyDefault ^AerospikeClient (get-client db ""))]
+                       (set! (.totalTimeout p) operation-timeout-ms)
+                       p)
+         k (str "__health__" (rand-int Integer/MAX_VALUE))
+         v (rand-int Integer/MAX_VALUE)
+         ttl (min 1 (int (/ operation-timeout-ms 1000)))
```

Comment:
should be `(int (Math/ceil (/ operation-timeout-ms 1000)))`
otherwise timeout is wrong on not whole seconds